tests/Resources/Page/LoginPage.robot

```diff
@@ -2,6 +2,7 @@
 Resource  OCPDashboard/OCPDashboard.resource
 Library   DebugLibrary
 Library   JupyterLibrary
+Library     SeleniumLibrary
```

Comment:
is it needed here?

Reply:
No not needed. I have removed it and will update the PR.